### ISSUE TITLE
I'm not a sybil attacker!

### ISSUE THREAD

**Issue by hebelubes** [hop-protocol/hop-airdrop#645]
I saw my address at "connectionBlacklist" with 9 other address. Surely they are not mine. 0x200998f1c6bb56b100752C9334Dd445c7d0740c3 (bthdbitcoin.eth) Just this one is mine and I had more than 1000 tx in 4 years. Also some of them is disperse txs from daos, VC's and has a lot of connection but this does not means all the address which I have tx is mine.

I dont care your airdrop but I'm not a sybil attacker. So check it again and delete my address from your list.

# Related Addresses
  0x200998f1c6bb56b100752c9334dd445c7d0740c3
  0x2e91728af3a54acdced7938fe9016ae2cc5948c9
  0x21bfd89ff9980e9ceef65812f9f739809e9639f6
  0x22708b691d0e4f67ee332cbd9e46f2a5a6420e17
  0x7a453ced24f502aa2cf5fbd8dffbdfb1dc0a4129
  0x348364ab7b30c4e634ae7aff693a63f49fdbff7a
  0xb4366f9a91c41acecb9af23097df826bb36246d8
  0x21e0ca21f517a26db49ec8fcf05fceabbabe98fa
  0x4800d8925c236bb779ae58a03c1999b1620af6e7
  0xc26ec0826a6186d3fcc7c8ea01bff19327b8d957
  0x320dda18c591787196d417c52ca939e010d02ad3

**Comment by Gwaram455** [comment]
Is like you have gotten 3250ARB tokens from Arbitrum airdrop despite being marked as sybil.

**Comment by hebelubes** [comment]
> 0xb4366f9a91c41acecb9af23097df826bb36246d8

Yes, I got many drops from many projects. Still hodling uni, op, arb etc.
